docs/chat/01-build/02-Develop-Initialize-User-Alt.mdx

```diff
@@ -22,6 +22,8 @@ import Details from '@theme/Details';
 
 ## Initialize user API
 
```

Comment:
This signature request can be bypassed by supplying the user's `decryptedPGPPrivateKey` in the options object. The `decryptedPGPPrivateKey` for a user can be found in the Push Profile Object, which is returned by `PushAPI.initialize`.

This part should be a note rather than here to suggest the idea that it is optional